README.md

```diff
@@ -1,12 +1,41 @@
 # lc3-vm-rust
 
-## UNDER CONSTRUCTION
-
 Basic implementation of the [LC3](https://www.jmeiners.com/lc3-vm/) virtual machine in Rust. An educational computer architecture, it has a simplified instruction set compared to x86.
 
+It supports the following instructions:
+
+- BR
+- ADD
+- LD
+- ST
+- JSR
+- JSRR
+- AND
+- LDR
+- STR
+- NOT
+- LDI
+- STI
+- JMP
+- LEA
+- TRAP
+
```

Comment:
I like the supported instructions list approach. To take it to the next level, you can use a two-column table (**Instruction** vs **Supported**) using emojis (✅ and ❌) if the instruction is supported or not. As it is written right now, I can't know if the supported instructions from the list are all the instructions that need to be supported.